packages/sdks-tests/src/e2e-tests/hit-content-api.spec.ts

```diff
@@ -40,4 +41,41 @@ test.describe('Get Content', () => {
     expect(await req!.postDataJSON()).toEqual({ test: 'test' });
     expect(req!.method()).toBe('POST');
   });
+  test('fetch symbol with query.id', async ({ page, sdk }) => {
+    test.skip(!excludeGen1(sdk));
+
+    let x = 0;
+    let headers;
+
+    const urlMatch = /https:\/\/cdn\.builder\.io\/api\/v3\/content\/symbol/;
+
+    const urls: string[] = [];
+
+    await page.route(urlMatch, route => {
+      x++;
+      headers = route.request().headers();
+
+      const url = new URL(route.request().url());
+      urls.push(url.href);
+      return route.fulfill({
+        status: 200,
+        json: {
+          results: [x === 0 ? FIRST_SYMBOL_CONTENT : SECOND_SYMBOL_CONTENT],
+        },
+      });
+    });
+
+    await page.goto('/get-content-with-symbol', { waitUntil: 'networkidle' });
+
+    await expect(x).toBeGreaterThanOrEqual(2);
+
+    urls.forEach(url => {
+      expect(url).toContain('query.id=');
```

Comment:
can we check that the value exists for sure, just to be safe?

```suggestion
      expect(url).toContain('query.id=foobar');
```

whatever `foobar` actually is